tests/Doctrine/Tests/OrmFunctionalTestCase.php

```diff
@@ -791,7 +791,7 @@ protected function onNotSuccessfulTest(\Throwable $e)
             foreach ($last25queries as $i => $query) {
                 $params = array_map(
                     function($p) {
-                        if (is_object($p)) return get_class($p); else return var_export($p, true);
+                        if (is_object($p)) return get_class($p); return var_export($p, true);
```

Comment:
Although things work fine, this bit is quite unreadable... could you please modify this to at least use a ternary expression?

Reply:
Pushing this change :man_technologist: